csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/FileContent.cs

```diff
@@ -119,6 +147,23 @@ private void DoInitialize()
                                 IsGroupMatch(line, ProjectSdk(), "Sdk", "Microsoft.NET.Sdk.Web") ||
                                 IsGroupMatch(line, FrameworkReference(), "Include", "Microsoft.AspNetCore.App");
                         }
+
+                        // Determine if implicit usings are used.
+                        if (!useImplicitUsings)
```

Comment:
It would be great, if we could also add a unittest (there is a unit test file for `FileContent.cs`).

Reply:
Nit: Maybe consider to use the short-circuit semantics of OR (I just realized I haven't done so in the lines above either)
```csharp
useImplicitUsings = useImplicitUsings || line.Contains(....
```